src/components/footer/NetworkStatus.tsx

```diff
@@ -0,0 +1,118 @@
+import { FC, useMemo } from 'react'
+import {
+  cn,
+  Dialog,
+  DialogBody,
+  DialogCloseDefault,
+  DialogContent,
+  DialogOverlay,
+  DialogTrigger,
+  Popover,
+  PopoverContent,
+  PopoverTrigger
+} from 'gfx-component-lib'
+// import { Circle } from '@/components/common/Circle'
+import { FooterItem, FooterItemContent } from '@/components/footer/FooterItem'
+import { Circle } from '@/components/common/Circle'
+import useNetworkStatus from '@/hooks/useNetworkStatus'
+import useBreakPoint from '@/hooks/useBreakPoint'
+const STATUS = {
+  CONGESTED: `The Solana network is very busy right now, your transactions might  take longer and sometimes they
+   might not go through at all.`,
+  DEGRADED: 'The Solana network is degraded, please be careful as most of your transactions might fail.',
+  NORMAL: 'The Solana network is running at normal speeds, expect normal to fast transactions.',
+  UNKNOWN: 'The Solana network status is unknown, please check back later.'
+}
+const NetworkStatus: FC = () => {
+  const { status } = useNetworkStatus()
+  const { isMobile } = useBreakPoint()
+  const { networkStatus, textColor, bgColor, description } = useMemo(() => {
+    switch (status) {
+      case 0:
+        return {
+          networkStatus: 'Normal',
+          textColor: 'text-background-green',
+          bgColor: 'bg-background-green',
+          description: STATUS.NORMAL
+        }
+      case 1:
+        return {
+          networkStatus: 'Degraded',
+          textColor: 'text-background-yellow',
+          bgColor: 'bg-background-yellow',
+          description: STATUS.DEGRADED
+        }
+      case 2:
+        return {
+          networkStatus: 'Congested',
+          textColor: 'text-background-red',
+          bgColor: 'bg-background-red',
+          description: STATUS.CONGESTED
+        }
+      default:
+        return {
+          networkStatus: 'Unknown',
+          textColor: 'text-background-green',
+          bgColor: 'bg-background-green',
+          description: STATUS.UNKNOWN
+        }
+    }
+  }, [status])
+  const component = useMemo(() => {
```

Comment:
The more ok pattern is to make this component in separate file. As for now, the component is bulky and have several responsibilities